Apps/WebClient/src/ClientApp/src/models/bannerError.ts

```diff
@@ -4,4 +4,11 @@ export default interface BannerError {
     detail: string;
     errorCode: string;
     traceId?: string;
+    source: string;
```

Comment:
@MikeLyttle I still need to remove the "details,errorCode" when I merge your changes?

Reply:
Yes, you should remove `description`, `detail`, and `errorCode`.  You'll also have to remove them from 2 functions in `utility\errorTranslator.ts` as well once you do that.